www/js/coolhole-util.js

```diff
@@ -1129,6 +1129,40 @@ applyAutoResizing();
 setupAutoHideUserlist();
 applyAutoHideUserlist();
 
+/**
+ * Constant for setting MOTD visibility in local storage.
+ */
+const HIDE_MOTD_STORAGE_NAME = "hide_motd";
+const motd = document.getElementById("motdwrap");
+
+function toggleHideMotd() {
+    const motdDisplay = window.getComputedStyle(motd).display;
+
+    if (motdDisplay !== "none" && window.localStorage.getItem(HIDE_MOTD_STORAGE_NAME) === "false") {
+      $("#motdwrap").hide();
+      $("#motd").hide();
+      window.localStorage.setItem(HIDE_MOTD_STORAGE_NAME, "true");
+    } else {
+      $("#motd").show();
+      $("#motdwrap").show();
+      window.localStorage.setItem(HIDE_MOTD_STORAGE_NAME, "false");
+    }
+
```

Comment:
concern(non-blocking): got a bug, once, where by toggling the motd while it was minimized, it expanded it and set the local storage flag to `true`. can't reproduce it though so take that for what you will.

nit: thinking about it, i don't think it's all that necessary to bother with checking the current display status of the wrapper. 
worst case scenario, you attempt to show or hide something that already has that state, respectively.
naively, i feel like this is simpler but lmk if i'm missing something
```suggestion
  const isHidden = localStorage.getItem(HIDE_MOTD_STORAGE_NAME) === "true";
  localStorage.setItem(HIDE_MOTD_STORAGE_NAME, !isHidden);
  applyHideMotd();
```